presentation/src/main/java/com/example/presentation/adapter/FollowerDetailAdapter.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.presentation.adapter
+
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+import com.example.presentation.R
+import com.example.presentation.databinding.ItemFollowerUserBinding
+import model.FollowerInfoResponse
+
+
+class FollowerDetailAdapter: ListAdapter<FollowerInfoResponse, FollowerDetailAdapter.ViewHolder>(
+    diffUtil
+) {
+
+    private lateinit var onItemClickListener: OnItemClickListener
+
+    fun followerItemClickListener(listener: OnItemClickListener){
+        this.onItemClickListener = listener
+    }
```

Comment:
```suggestion
    private var onItemClickListener: OnItemClickListener? = null

    fun followerItemClickListener(listener: OnItemClickListener){
        this.onItemClickListener = listener
    }
```

lateinit var는 생성자처럼 초기에 무조건 초기화되는 변수가 아니면 안쓰는 것이 좋을 것 같다는 아주 개인적인 생각이 있어요!
?: , ? 처럼 언어적으로 처리하기 쉽지 않고, 클래스 파일이 복잡해지고 참조하는 곳이 많아지면 어느 시점에 초기화되어있는지 파악하기 어려워서요 :)
참고 부탁드립니다 🤺

Reply:
호오 감사합니다!